content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -94,7 +92,17 @@ run the `kfctl apply` command:
 ```bash
 mkdir -p ${KF_DIR}
 cd ${KF_DIR}
-kfctl apply -V -f ${CONFIG_URI}
+
+# Recommended: Download the config file and change the default login credentials.
+wget -O kfctl_existing_arrikto.yaml $CONFIG_URI
```

Comment:
I see why they should do this. We could just instruct them to run `kfctl build`, then change the credentials, then run `kfctl apply`. But if this requirement is going away (default Service type for the Istio Ingressgateway will no longer be NodePort) then it's OK to leave this as is.

Reply:
I thought about it as well, but I believe it's not necessary to run build first, since we're changing the KfDef and not the generated packages.
This requirement should go away in the near future.

Reply:
Sounds good.